ui/src/components/media/calling.rs

```diff
@@ -490,6 +494,11 @@ fn PendingCallDialog(cx: Scope<PendingCallProps>) -> Element {
             }
         };
     }
+    let alive = use_ref(cx, || Arc::new(AtomicBool::new(false)));
```

Comment:
because of this being atomic, does this have to set behind a `use_ref`? Also, where is `AtomicBool` being set at for it to trigger in the condition?

Reply:
it might be better to use a `UseRef` instead of an `Arc`, to ensure that if somehow the Element re-renders, it doesn't make a new `AtomicBool`.